app/src/main/java/org/fossasia/openevent/app/module/event/dashboard/EventDashboardFragment.java

```diff
@@ -143,7 +143,7 @@ private void setupRefreshListener() {
 
     @Override
     public void showProgress(boolean show) {
-        ViewUtils.showView(binding.progressBar, show);
+        ViewUtils.showView(binding.progressView, show);
```

Comment:
Is it necessary to rename this? It is a progress bar after all. Also, we can create a common layout and just import it in each one